### DISCUSSION TITLE
Spellcheck is not working in CEF Minimal Example

### DISCUSSION THREAD

**Discussion by a2bhijeet** [cefsharp/CefSharp#4533] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

110.0.300

### Operating System

Windows 10

### Architecture

x64

### .Net Version

4.6.2

### Implementation

WinForms

### Reproduction Steps

1. Setup a machine where download is not allowed in C: directory
2. Set language to 'en-US'. (This can be set to any value for which we want spell check to work)
3. Open the CEF Minimal Example
4. Open any website. Check if spell check is working.

### Expected behavior

Spell check should work

### Actual behavior

Spell check is not working

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Converted to discussion as the spellchecking feature is implemented entirely in `CEF/Chromium`.

> CefSharp Version
> 110.0.300

Please don't open issues for old unsupported versions. Always test with the current supported version to determine if your issue has been resolved (M114 is the current at time of writing).

> 1. Setup a machine where download is not allowed in C: directory

It's not entirely clear what you mean by this. Are you saying `CEF` is attempting to write to the root of `C:`?

The dictionary is downloaded at runtime as `CEF` does not yet support using the windows spell check, https://github.com/chromiumembedded/cef/issues/3055

There are some changes to where CEF downloads data coming as part of https://github.com/chromiumembedded/cef/issues/3511#issuecomment-1578699979

> Does this problem also occur in the CEF Sample Application
> Not Tested

For features like spellchecking it's importing this testing is performed.
